### PR TITLE
Add io.github.mimbrero.WhatsAppDesktop app

### PR DESCRIPTION
**Pull request by mimbrero**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/mimbrero/whatsapp-desktop-linux
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by mimbrero** [comment]
Bot, build

**Comment by mimbrero** [comment]
Bot, build io.github.mimbrero.WhatsAppDesktop

**Comment by mimbrero** [comment]
bot, build io.github.mimbrero.WhatsAppDesktop

**Comment by hfiguiere** [comment]
build failed because of appdata:
```
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
```

**Comment by mimbrero** [comment]
> build failed because of appdata:
> ```
> • tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
> ```

I was going to ask about this. As this is a messaging app, you can obviously receive violent content, pictures that include drugs or nudity... what do I need to specify? Everything or just that this is a full messaging app in the social section? 🙂

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```
[the remark this answers is not in the document]
Comment (mimbrero) [reply flathub.json:2]: True, I don't know why but I assumed that electron-builder built only for x64 (????)

**Comment by mimbrero** [comment]
bot, build io.github.mimbrero.WhatsAppDesktop

**Comment by mimbrero** [comment]
GG, all done then?

**Comment by mimbrero** [comment]
Thanks for your time @hfiguiere and @barthalion 😄

**Pull request closed without merging**